django/contrib/auth/views.py

```diff
@@ -446,12 +454,25 @@ def dispatch(self, *args, **kwargs):
         self.validlink = False
         self.user = self.get_user(kwargs['uidb64'])
 
-        if self.user is not None and self.token_generator.check_token(self.user, kwargs['token']):
-            self.validlink = True
-        else:
-            return self.render_to_response(self.get_context_data())
-
-        return super(PasswordResetConfirmView, self).dispatch(*args, **kwargs)
+        if self.user is not None:
```

Comment:
Probably not the most elegant, but I think quite readable to understand this token logic.